stage #1/module #6. IO, NIO/lesson #1. IO/content.md

```diff
@@ -0,0 +1,127 @@
+# Java I/O (java.io)
```

Comment:
Sorry, but module name is **IO, NIO**. Will you be doing another pull request for NIO?

Reply:
Yes, I'll do this in another PR.